strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/nuget/file/ContentFile.java

```diff
@@ -0,0 +1,90 @@
+package org.carlspring.strongbox.nuget.file;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(
```

Comment:
Cosmetic: Chopped parameters should start on the same line as the function/annotation and be formatted like this:
```
@XmlRootElement(name = "files",
                namespace = "http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd")
```